Makefile

```diff
@@ -668,50 +658,26 @@ isti%.yaml: $(HELM) $(HOME)/.helm
 		--values install/kubernetes/helm/istio/values-$@ \
```

Comment:
were you suggesting in your proposal to have these values in releases directory ?